### PR TITLE
Contacts feature updated to retrieve geo location to be rendered in M…

### PR DESCRIPTION
**Pull request by kyoon0**
…yTrip

## Overview

**Issue Type**

- [x] Bug
- [x] Feature
- [ ] Tech Debt

**Description**
Contacts
- Updated StartTrip button logic to retrieve geo location data and render on My Trip
- Minor UI edits on Contacts

**Ticket Item**
[FE] Contacts
[FE] My Trips

### CODE REVIEW

File: TS_src/client/src/App.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Login from './components/pages/Login';
 import Registration from './components/pages/Registration';
 import Dashboard from './components/pages/Dashboard';
 import ChatPage from './components/ChatPage';
-import { User } from './components/types';
+import { User, Trip } from './components/types';
 import socket from './main';
 import toast, { Toaster } from 'react-hot-toast';
```
Comment (YouruiR) [inline TS_src/client/src/App.tsx]: Is the import `toast` being used? I see that Toaster is for sure, but wasn't sure about toast. I noticed there were other unused imports as well such as `{ Component }` from `'react'` and `{ Link }` from `react-router-dom`. Are those safe to take out?

File: TS_src/client/src/App.tsx [hunk 2]
```diff
@@ -24,17 +24,6 @@ function App() {
...
-		active: boolean;
-		start_timestamp: string;
-		start_lat: string;
-		start_lng: string;
-		sos_timestamp: string;
-		sos_lat: string;
-		sos_lng: string;
-	}
-
 	interface TripProps {
```
Comment (YouruiR) [inline TS_src/client/src/App.tsx]: Could we also abstract the `TripProps` interface out? Even though it is only used here, I think it makes sense to just have all our types in one place and never have them in the frontend code that's run.

File: TS_src/client/src/App.tsx (hunk 1 above)
Comment (kyoon0) [reply TS_src/client/src/App.tsx]: Removed

File: TS_src/client/src/App.tsx (hunk 2 above)
Comment (kyoon0) [reply TS_src/client/src/App.tsx]: Extracted TripProps and migrated to Types.tsx. Will need to update all other components in later PRs for best practice.

**Review by YouruiR — APPROVED** [review]

**Review by YouruiR — APPROVED** [review]

**Review by tcchu — APPROVED** [review]
approved

**Pull request merged**